tests/test_jupyter_tutorial.py

```diff
@@ -0,0 +1,72 @@
+from cookiecutter.main import cookiecutter
+import os
+import filecmp
+import shutil
+import filecmp
+
+
+
+def download_tutorial():
+    try:
+        cookiecutter(
+            "https://github.com/SebChw/art_template.git",
+            no_input=True,
+            extra_context={"project_name": "mnist_tutorial", "author": "test", "email": "test"},  # Pass the project_name to the template,
+            checkout="mnist_tutorial_cookiecutter",  # Use the latest version of the template
+        )
+    except Exception as e:
+        print("Error while generating project using Cookiecutter:", str(e))
+        raise e
+    return 0
```

Comment:
This function shouldn't return anything

Reply:
Right, i was just checking sth and forgot to remove it